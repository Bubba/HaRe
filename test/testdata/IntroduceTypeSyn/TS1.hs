module TS1 where

type Foo = (String, Int)

getInt :: (String, Int) -> Int
getInt (_, i) = i
