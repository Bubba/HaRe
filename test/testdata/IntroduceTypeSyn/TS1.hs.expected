module TS1 where

type Foo = (String, Int)

getInt :: Foo -> Int
getInt (_, i) = i
